platform/state/platform_state.proto

```diff
@@ -159,6 +159,11 @@ message ConsensusSnapshot {
      * A "consensus" timestamp.<br/>
      * The consensus timestamp of this snapshot.
      * <p>
+     * Depending on the context this timestamp may have different meanings:
```

Comment:
I feel like this is an implementation detail, not sure if it needs to be documented here. it should be enough to say that this is the consensus time at the end of the round

Reply:
@lpetrovic05 I did it by Kelly's request. Here is the context - https://github.com/hashgraph/hedera-services/pull/13730#discussion_r1647552595 
I made this change to keep it in sync with the design proposal.